my-vars/config-sample-do-not-edit.yml

```diff
@@ -64,10 +64,43 @@
 # - remove ipsets
 # block_tor_exit_nodes: True
 
+###############################################################################
+# HashiCorp Vault
+#
+# HashiCorp Vault can be used for managing secrets with Ansible, such as
+# server config files and login information.
+#
+# When you install HashiCorp Vault you can create one or more storage backends
+# within where your secrets will be stored. Following variable should be set to
+# your storage backend directory:
+hashicorp_vault_storage_backend_dir: "kv/data"
```

Comment:
Since all of these are in the `webportal.yml` file, should these be commented out by default? Then the instructions would be, to only uncomment and use these fields if the default paths/names are not going to be used.

Reply:
Commented out.

`hashicorp_vault_storage_backend_dir` is one of the exceptions, because the user should explicitly set its value.